package.json

```diff
@@ -37,64 +36,60 @@
     ]
   },
   "dependencies": {
-    "@arcgis/core": "^4.31.6",
+    "@arcgis/core": "^4.32.10",
     "@heroicons/react": "^2.2.0",
-    "@tanstack/react-query": "^5.63.0",
-    "@tanstack/react-table": "^8.20.6",
-    "@ugrc/layer-selector": "^6.2.12",
-    "@ugrc/tailwind-preset": "^1.1.0",
-    "@ugrc/utah-design-system": "^1.18.2",
-    "firebase": "^11.1.0",
+    "@tanstack/react-query": "^5.76.1",
+    "@tanstack/react-table": "^8.21.3",
+    "@ugrc/layer-selector": "^6.4.0",
+    "@ugrc/tailwind-preset": "^1.2.0",
+    "@ugrc/utah-design-system": "^1.27.1",
+    "@ugrc/utilities": "^3.0.4",
+    "firebase": "^11.8.0",
     "immer": "^10.1.1",
-    "ky": "^1.7.4",
+    "ky": "^1.8.1",
     "lodash.debounce": "^4.0.8",
-    "react": "^19.0.0",
-    "react-dom": "^19.0.0",
-    "react-error-boundary": "^5.0.0",
+    "lucide-react": "^0.511.0",
+    "react": "^19.1.0",
+    "react-aria": "^3.40.0",
+    "react-dom": "^19.1.0",
+    "react-error-boundary": "^6.0.0",
+    "react-stately": "^3.38.0",
+    "tailwind-merge": "^3.3.0",
```

Comment:
this dep needs to stay at version 2. Please doc that, tailwindcss, and tailwindcss-react-aria-components can't be updated until we migrate the design system to tailwind 4.